source/ShoutrApp/src/app/peer-icon/peer-icon.component.scss

```diff
@@ -0,0 +1,7 @@
+.row{
+    display: flex;
+    .space{
+        height: 10px;
+        width: 10px;
```

Comment:
I would like to (eventually) refactor this component so the implementer decides the size of this component. 
In this case connected would set the height/width of peer-icon and the spaces would evenly distribute that space.

Reply:
done